test/Microsoft.TemplateEngine.TestHelper/PackageManager.cs

```diff
@@ -137,6 +143,10 @@ public string PackNuGetPackage(string projectPath)
             if (string.IsNullOrWhiteSpace(version))
             {
                 (source, packageMetadata) = await GetLatestVersionInternalAsync(identifier, packagesSources, log, cancellationToken).ConfigureAwait(false);
+                if (minimumVersion is not null && packageMetadata.Identity.Version < minimumVersion)
```

Comment:
Tests will still fail here, but at least it will be easier for us to understand the reasoning, right?

Reply:
correct. I also added nuget org as source, I hope that one won't fail :)